src/builtin/cd_command.ts

```diff
@@ -5,9 +5,29 @@ import { IRunContext, ITabCompleteContext } from '../context';
 import { GeneralError } from '../error_exit_code';
 import { ExitCode } from '../exit_code';
 import { ITabCompleteResult, PathType } from '../tab_complete';
+import { BooleanArgument } from '../argument';
```

Comment:
`'../argument'` is already imported a few lines above, so add `BooleanArgument` to that.